apps/emqx_ft/include/emqx_ft.hrl

```diff
@@ -0,0 +1,22 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2023 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+
+-ifndef(EMQX_FT_HRL).
+-define(EMQX_FT_HRL, true).
+
+-define(FT_RESPONSE_TOPIC, <<"$file/response">>).
```

Comment:
Few thoughts.
1. Wouldn't be more predictable / scalable for a client to subscribe to `$file/response/{{fileId}}`, so they won't be overwhelmed with feedback related to **all** transfers? Also should be more future-proof from the security standpoint.
2. Seems unfortunate that it collides with command topics space, might lead to some "messiness" (e.g. you can subscribe to `$file/response` but not to `$file/{{fileId}}`, you can still use `"response"` as fileId but feedback will look somewhat confusing). Maybe `$fileack/...` or something like that would be better?
3. Nit. It's not exactly a _responses_ that are going here (since they're entirely optional), maybe it's _status_ or _feedback_?

Reply:
The first variant I implemented was actually `$file/{{fileId}}/response`, but this scheme doesn't work for malformed commands.
We decided to make the default response topic `$file/{{clientId}}/response` during the last discussion.
`{{fileId}}` is not included so that the client should not resubscribe on each file transfer.

Reply:
> We decided to make the default response topic `$file/{{clientId}}/response` during the last discussion.
> `{{fileId}}` is not included so that the client should not resubscribe on each file transfer.

Good point. Although I'm still wondering: given the client can subscribe to a wildcard anytime, wouldn't it be strictly more flexible to publish to `$file/{{clientId}}/{{fileId}}/response` then?